packages/mux-audio/package.json

```diff
@@ -53,7 +53,7 @@
   "devDependencies": {
     "@mux/test-esm-exports": "0.1.0",
     "@open-wc/testing": "^3.0.3",
-    "@web/dev-server-esbuild": "^0.2.16",
+    "@web/dev-server-esbuild": "^0.3.2",
```

Comment:
question(non-blocking): Why did we upgrade the version in this PR? Are there changes we should be aware of?

Reply:
yes, this was required because the older plugin did not support the inline `type` keyword which we mentioned yesterday. e.g. `import { type ValueOf } from './types'`